### PR TITLE
Install headers, pkgconfig file, library

### PR DESCRIPTION
**Pull request by nirbheek**

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -123,27 +143,16 @@ _files = [
   config_h,
 ]
 
-if cc.get_id() == 'msvc'
```
Comment (sarum9in) [inline meson.build:126]: Why is it safe to remove this now and wasn't safe before?

**Review by sarum9in — CHANGES_REQUESTED** [review]

File: meson.build (hunk 1 above)
Comment (dcbaker) [reply meson.build:126]: IIRC (and it's been a while) expat versions the library on windows when built with msvc, but not when built with mingw (or maybe it's the other way around?). Not sure if it's a good idea to remove that or not.

Comment (nirbheek) [reply meson.build:126]: The `install:` option was always incorrect (there is no reason to not install static libraries too).

I just verified that when built with `cmake -G "NMake Makefiles"` inside a Visual Studio command prompt, the output is `expat.dll` without an soversion suffix, so this should match what the cmake build files output.

Comment (sarum9in) [reply meson.build:126]: Okay, let's approve that given that @nirbheek verified it works as intended.

**Pull request merged**